cpp/include/ray/api/generated/actor_call.generated.h

```diff
@@ -0,0 +1,25 @@
+// TODO(Guyang Song): code generation
```

Comment:
I tend to implement the code generation in next PR which contains cluster mode. Because I think the generated code has not been finalized. Maybe a big change. I'm not ensure.